_docs/overview/key-concepts.md

```diff
@@ -34,6 +34,12 @@ This is analogous to storing coins in a padlocked box, with multiple keys and
 padlocks which open in different combinations. See the [Multisignature Security](/docs/overview/multi-signature-security)
 section for more detail.
 
+## Redeem script
+
+Multisignature requires each of the multiple private keys to be accompanied by a
+redeem script. This script, each of which is identical, defines how the keys
```

Comment:
Shouldn't this read "all of which are identical"? "Each is identical" seems to be missing something...each is identical to what?